apps/publish/enqueue/enqueue_service.py

```diff
@@ -231,6 +231,9 @@ def resend(self, doc, subscribers):
             self._resend_to_subscribers(doc, wire_subscribers, subscriber_codes)
 
         if len(digital_subscribers) > 0:
+            if is_genre(doc, BROADCAST_GENRE):
+                raise SuperdeskApiError.badRequestError('Only wire subscribers can receive broadcast stories!')
```

Comment:
just not sure it should raise an error, because it might still send it to those wire subscribers. but not sure how a warning status code would look like